patch/cisco-acpi-spi-nor.patch

```diff
@@ -0,0 +1,30 @@
+commit baaac832b8a63756933f71ead22ce2bb9ed98eb0
+Author: balsup <balsup@contoso.com>
+Date:   Thu Aug 13 23:57:13 2020 +0000
+
+    Support ACPI table match
```

Comment:
Please create the patch file with `git format-patch` and add a commit message body explaining the problem and fix, and includes a Signed-off-by line.